<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibrator</class>
 <widget class="QMainWindow" name="Calibrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>93</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:center;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="html">
     <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'HelveticaNeueCyr'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pBtn_Channel_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Канал 1</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
=======
    <property name="text">
     <string>Канал 1</string>
>>>>>>> origin/master
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
<<<<<<< HEAD
     <bool>false</bool>
=======
     <bool>true</bool>
>>>>>>> origin/master
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupChannel</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="pBtn_Channel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Канал 2</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
=======
    <property name="text">
     <string>Канал 2</string>
>>>>>>> origin/master
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupChannel</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="pBtn_Channel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Канал 3</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
=======
    <property name="text">
     <string>Канал 3</string>
>>>>>>> origin/master
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupChannel</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="pBtn_Channel_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Канал 4</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
=======
    <property name="text">
     <string>Канал 4</string>
>>>>>>> origin/master
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupChannel</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="pBtn_Channel_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Канал 5</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
=======
    <property name="text">
     <string>Канал 5</string>
>>>>>>> origin/master
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupChannel</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="pBtn_Channel_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Канал 6</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
=======
    <property name="text">
     <string>Канал 6</string>
>>>>>>> origin/master
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupChannel</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_15">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_17">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_19">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_20">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_21">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_24">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="R100">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
=======
>>>>>>> origin/master
    <property name="text">
     <string>100</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupResistance</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="R166">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
=======
>>>>>>> origin/master
    <property name="text">
     <string>166</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupResistance</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="R200">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
=======
>>>>>>> origin/master
    <property name="text">
     <string>200</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupResistance</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="R300">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
<<<<<<< HEAD
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
=======
>>>>>>> origin/master
    <property name="text">
     <string>300</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GroupResistance</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Volts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>375</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 step - get 4 dot, 2nd step - calc channel?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
<<<<<<< HEAD
  <buttongroup name="GroupChannel">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
=======
  <buttongroup name="GroupChannel"/>
>>>>>>> origin/master
  <buttongroup name="GroupResistance"/>
 </buttongroups>
</ui>
